test/TeenyStatistics.ts

```diff
@@ -158,9 +158,8 @@ describe('TeenyStatistics', () => {
     it('should be read-only', () => {
       const t = new TeenyStatistics();
       assert.throws(() => {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-        // @ts-ignore
-        t.counters = {concurrentRequests: 99};
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (t as any).counters = {concurrentRequests: 99};
```

Comment:
I don't think our linter likes it, but I usually add a `;(t as any)` to a line like this:

```js
function f (arg) { console.info(arg) }
f
('hey');
```